src/components/GoodsList.tsx

```diff
@@ -0,0 +1,11 @@
+import { Good } from './Good';
+
+type GoddsListProps = {
+  goods: string[]
+};
+
+export const GoodsList = ({ goods }: GoddsListProps) => (
+  <ul>
```

Comment:
You are using semantic HTML. Great job!
Using `ul` and `li` to represent a list of items will help your code and also improve the user experience because the browser will understand better the elements on the page and it also improves the page accessibility.